templates/_layouts/docs.html

```diff
@@ -163,6 +163,7 @@
       window.Prism = window.Prism || {};
       Prism.manual = true;
     </script>
+    <script src="/static/js/dist/liveDemoBox.js" type="module"></script>
```

Comment:
Does this not work with `version_static` module?

Reply:
I changed it to use `versioned_static`.